src/components/breadcrumb/daikin-breadcrumb.ts

```diff
@@ -40,124 +39,44 @@ export class DaikinBreadcrumb extends LitElement {
     ${unsafeCSS(tailwindStyles)}
 
     :host {
-      display: flex;
-      align-items: center;
+      padding-top: 0.25rem;
+      padding-bottom: 0.25rem;
     }
   `;
 
-  private resizeObserver = new ResizeObserver(() => {
-    this._omit();
-  });
-
-  // get daikin-breadcrumb-item from shadow root
-  @queryAssignedElements()
-  private _slottedDaikinBreadCrumbItems!: Array<DaikinBreadcrumbItem>;
-
-  // get div element from daikin-breadcrumb
-  private _divWrapRef: Ref<HTMLDivElement> = createRef();
-
-  /**
-   * Whether the last breadcrumb item should have trailing slash.
-   */
-  @property({ type: Boolean, reflect: true, attribute: "trailing-slash" })
-  trailingSlash = false;
-
   /**
-   * Specifies overflow.
-   * If `ellipsis`, some breadcrumb items will be omitted when the total width exceeds the container width.
+   * Whether or not to change the color of visited links.
    */
-  @property({ type: String, reflect: true })
-  overflow: "visible" | "ellipsis" = "visible";
+  @property({ type: Boolean, reflect: true, attribute: "show-visited" })
+  showVisited = false;
 
-  private _expandedContentWidth: number = 0;
-
-  private omitted: boolean = false;
-
-  get _isEllipsis() {
-    return this.overflow === "ellipsis";
-  }
-
-  private _omit() {
-    // remove items and add omission if daikin-breadcrumb is to long
-    const shouldOmit =
-      this._isEllipsis && this.offsetWidth < this._expandedContentWidth;
-    if (this.omitted === shouldOmit) {
-      return;
-    }
-    this.omitted = shouldOmit;
-
-    const daikinBreadCrumbItems = this._slottedDaikinBreadCrumbItems;
-    for (const [index, item] of daikinBreadCrumbItems.entries()) {
-      let mode = "normal";
-      if (shouldOmit) {
-        if (index === 0 || index >= daikinBreadCrumbItems.length - 2) {
-          continue;
-        } else if (index === 1) {
-          mode = "ellipsis";
-        } else {
-          mode = "hidden";
-        }
-      }
-      item.hidden = mode === "hidden";
-      item.variant = mode === "ellipsis" ? "ellipsis" : "normal";
-    }
-  }
+  @queryAssignedElements({ selector: "daikin-breadcrumb-item" })
+  private readonly _items!: readonly DaikinBreadcrumbItem[];
 
   private _updateBreadcrumbs() {
-    const daikinBreadCrumbItems = this._slottedDaikinBreadCrumbItems;
-    for (const [index, item] of daikinBreadCrumbItems.entries()) {
-      const isLast = index === daikinBreadCrumbItems.length - 1;
-      item.trailingSlash = !isLast || this.trailingSlash;
-      item.last = isLast;
-    }
-  }
+    const items = this._items;
 
-  private _handleChange() {
-    this._updateBreadcrumbs();
+    items.forEach((item, index) => {
+      item.variant = index === items.length - 1 ? "current" : "normal";
+      item.showVisited = this.showVisited;
+      item.showDivider = index < items.length - 1;
```

Comment:
Try not to use different expressions (`!==` and `<`) to mean the same thing.

```suggestion
      const isLast = index === items.length - 1;
      item.variant = isLast ? "current" : "normal";
      item.showVisited = this.showVisited;
      item.appendDivider = !isLast;
```